static/src/stylesheets/module/crosswords/_grid.scss

```diff
@@ -8,6 +8,10 @@
     fill: #ffffff;
 }
 
+.crossword__cell--intersecting {
+    fill: hsl(0, 0%, 80%);
```

Comment:
should really be one of our neutral colours: http://guardian.github.io/pasteup-palette/demo/